src/components/Testimonials/Testimonials.css

```diff
@@ -0,0 +1,48 @@
+.testimonials {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 90px 60px 50px;
+  gap: 10px;
+  color: #ffffff;
+}
+
+.testimonialsHead {
+  padding-bottom: 40px;
+}
+
+.testimonialsTitle {
+  text-align: center;
+  font-size: 52px;
+}
+
+.testimonialsItems {
+  display: flex;
+  flex-direction: row;
+}
+
+.testimonialCardDiv {
+  display: table;
+  align-items: center;
+  padding: 10px;
+}
+
+.testmonialCard {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 40px;
+  gap: 30px;
+  width: 430px;
+  background: #181818;
+  border: 1px solid #dedede;
+  border-radius: 10px;
+  margin-bottom: 20px;
+}
+
+.client {
+  display: flex;
+  flex-direction: row;
+  gap: 13px;
```

Comment:
`display:flex` is row by default.
use `align-items:center` to centralise the client avatar and details